src/extension.ts

```diff
@@ -212,6 +216,9 @@ export class Extension {
             const config = e.body as BrightScriptLaunchConfiguration;
             await docLinkProvider.setLaunchConfig(config);
             logOutputManager.setLaunchConfig(config);
+            if (config.remoteControlMode && (config.remoteControlMode === true || config.remoteControlMode?.activateOnSessionStart === true)) {
```

Comment:
Just check the `.activateOnSessionStart`.
Probably something like `config?.remoteControlMode?.activateOnSessionStart === true` since our DebugConfigurationProvider will have sanitized this to always be in this structure.